### DISCUSSION TITLE
May/15 Update

### DISCUSSION THREAD

**Discussion by corygalyna** [ZORALab/Hestia#105] [Q&A] [answered]
Update @hollowaykeanho?

**Comment by hollowaykeanho** [comment, marked as answer]
Current working:

https://github.com/ZORALab/Hestia/issues/79 - Already structured the packages and API to the first planned eversince getting rid of hestiaRS. At the moment finalizing the transformation and likely commit today or tomorrow. Issue can be pulled in from Phase 2 to Phase 1.2 and we no longer need to separately prepare the documentations just for Rust.

Once this is done, porting documentations shall be resumed.

**Comment by corygalyna** [comment]
Entire hestiaHUGO?! including renderers and outputs?

**Comment by hollowaykeanho** [comment]
Yeap. Technically speaking, my next commit can close Issue #79 with Hugo, Go, and Tinygo all share the same naming conventions and API.

**Comment by corygalyna** [comment]
That's good news. Carry on.
